guides/common/modules/con_creating-and-managing-roles.adoc

```diff
@@ -1,7 +1,42 @@
 [id="Creating_and_Managing_Roles_{context}"]
+ifeval::["{context}" == "admin"]
 = Creating and managing roles
+endif::[]
+ifeval::["{context}" == "planning"]
+= Defining role-based access control policies
+endif::[]
 
-{Project} provides a set of predefined roles with permissions sufficient for standard tasks, as listed in xref:Predefined_Roles_Available_in_{project-context}_{context}[].
-It is also possible to configure custom roles, and assign one or more permission filters to them.
+Users in {Project} can have one or more roles assigned.
+These roles are associated with permissions that enable users to perform specified administrative actions in {Project}.
 Permission filters define the actions allowed for a certain resource type.
-Certain {Project} plugins create roles automatically.
+
+{Project} provides a set of predefined roles with permissions sufficient for standard tasks.
+ifeval::["{context}" == "admin"]
+For a list of these roles, see xref:Predefined_Roles_Available_in_{project-context}_{context}[].
+endif::[]
+You can also configure custom roles.
+
+[NOTE]
+====
+One of the predefined roles is the *Default role*.
+{Project} assigns the *Default role* to every user in the system.
+By default, the *Default role* grants only a limited set of permissions.
+Be aware that if you add a permission to the *Default role*, every {Project} users will gain that permission.
+Assigning a different role to a user does not remove the *Default role* from the user.
+====
+
+The following types of roles are commonly defined within various {Project} deployments:
+
+Roles related to applications or parts of infrastructure::
+For example, roles for owners of {client-os} as the operating system as opposed to roles for owners of application servers and database servers.
+
+Roles related to a particular stage of the software lifecycle::
+For example, roles divided among the development, testing, and production phases, where each phase has one or more owners.
+
+Roles related to specific tasks::
+For example, role of a security manager or role of a license manager.
```

Comment:
_-Wpedantic_

> role of a license manager

If someone came to us and asked "I'm a license manager, the docs say a role can be created for me. How do I set it up?", would we be able to answer that?